App/ListJS.js

```diff
@@ -0,0 +1,103 @@
+var playListFromJS= JSON.parse(playlistsJSON);
+// console.log("PList:"+playListFromJS[0]);
+
+
+document.addEventListener("DOMContentLoaded", function(event) {
+
+
+  if( AccountService._instance.loggedUser !== null)
+  {
+      console.log("USER logged",  AccountService._instance.loggedUser);
+      document.getElementById('signInButton').style.visibility =  "hidden";
+      document.getElementById('logOutButton').style.visibility =  "visible";
+      document.getElementById('username').innerText ="Hello " + AccountService._instance.loggedUser.name + " !" ;
+  }
+  else {
+      console.log("USER NOT logged");
+      document.getElementById('logOutButton').style.visibility =  "hidden";
+      document.getElementById('username').innerText = "";
+  }
+
+    // /createPlayLists();
+
+  });
+
+//function Create playlists
+function createPlayLists(){
+  console.log(playListFromJS);
+  for (var i = 0; i < playListFromJS.length; i++) {
```

Comment:
map from json to Playlisg objects then do a second map from Playlist object to drawBigPlaylist(which should just return a DOM Node),  With the resulting collection of dom nodes add them to DOM